tensorflow/tools/pip_package/setup.py

```diff
@@ -123,7 +123,7 @@ def standard_or_nightly(standard, nightly):
     # current release version. These also usually have "alpha" or "dev" in their
     # version name.
     # These are all updated during the TF release process.
-    standard_or_nightly('tensorboard >= 2.11, < 2.12',
+    standard_or_nightly('tensorboard >= 2.12, < 2.13',
                         'tb-nightly ~= 2.12.0.a'),
```

Comment:
Since this is a PR on master, it should change nightly.

But as @nfelt said, this needs to wait until TB release